### ISSUE TITLE
Add create new query to the right-click context menu

### ISSUE THREAD

**Issue by MillTurnBr** [HeidiSQL/HeidiSQL#1036]
## Steps to reproduce this issue

1. Open a DB in HeidiSQL
2. Click on a table
3. Right-click

![image](https://user-images.githubusercontent.com/65592005/82300834-001bb900-997d-11ea-9eae-4e9bfb54a153.png)

3. There's no option to create a query for the selected table

## Current behavior
There's no option in the right-click menu to create a query for the selected table

## Expected behavior
Have two options to create two types of query when the user right-click a selected table:

Option 1. - Create a new query tab and populate it with a pre-defined basic **SELECT** query
Option 2. - Create a new query tab, **but empty**

## Possible solution

In Microsoft SSMS, such option do exist, as shown below:

**If the user right-click a table, the option to SELECT <n> columns is available**

![image](https://user-images.githubusercontent.com/65592005/82300633-b92dc380-997c-11ea-85f2-72c545c7bcb3.png)

**This opens a new query tab with a basic script from a pre-defined template**

![image](https://user-images.githubusercontent.com/65592005/82300657-c2b72b80-997c-11ea-8f0e-d78863a87ba8.png)

**The number of rows in the SELECT statement is configurable in the application options:**

![image](https://user-images.githubusercontent.com/65592005/82300673-c6e34900-997c-11ea-831b-e8dd86870784.png)

* **HeidiSQL version:** 
11.0.0.5985

**Comment by ansgarbecker** [comment]
Did you see that the "Data" tab in Heidi already does what you describe? The default limit is even configurable:
![grafik](https://user-images.githubusercontent.com/7986591/82305022-69232100-99bd-11ea-9ece-2a095d986c3a.png)

**Comment by MillTurnBr** [comment]
Yes, I noticed that but that's not used in a default SELECT query is it?

The options were just the cherry in the top of the cake... My original intent here was to add the "Create New Query" in the right-click context menu, and add two options for queries: A default "SELECT" query like in the pic above, based on a hardcoded template, and another one just to create a new empty query tab... Tks for your feedback anyway!

**Comment by ansgarbecker** [comment]
Well it's still a valid feature request. I just wanted to make sure you saw the data tab, and that Heidi behaves differently than SSMS.
But let's add these new context menu items in the "Create new" submenu.

**Comment by MillTurnBr** [comment]
Not trying to abuse here and if I ever request that it would be a new ticket...
But I just realized that the users could kind of point Heidi to a query templates folder with *.sql files with queries.... and then in the right-click menu we could have an option to "Open new query Window and insert a template"

Or define a template in the settings that could be inserted via right-click menu...

I think that by now you picture what I mean... Templates are great to speed up a DBA's work...

**Comment by ansgarbecker** [comment]
That template idea sounds nice. We should just define them here, I don't see the need to make them customizable from the beginning.
Related: HeidiSQL supports snippets, on the right part of a query tab. These do not have some placeholders, so they're not templates.

**Comment by ansgarbecker** [comment]
Done through #604 for v12.11
